tasks/main.yml

```diff
@@ -150,6 +169,22 @@
     - postfix
     - postfix-generic-table
 
+- name: configure header checks
+  template:
+    src: etc/postfix/header_checks.j2
+    dest: /etc/postfix/header_checks
+    owner: root
+    group: root
+    mode: 0644
+  when: postfix_header_checks is defined
+  with_items: "{{ postfix_header_checks }}"
```

Comment:
No need to `postmap` this file?

Reply:
afaik there's no need with regexp. And I only use it with regex.

Reply:
Does it do any harm?

Reply:
Running `postmap {{ postifx_header_checks_database_type }}:/etc/postfix/header_checks`

Reply:
That's what i just thought as well. However it actually does do harm, because postmap can only be run with the following file_types: btree, cdb, dbm, hash, fail, sdbm (see http://www.postfix.org/postmap.1.html). 
If it is run like you suggested, there will be an error: 
`postmap regexp:/etc/postfix/header_checks
postmap: fatal: unsupported dictionary type: regexp. Is the postfix-regexp package installed?`

While looking into it I found a thread at saltstack discussing this kind of issue: https://github.com/saltstack-formulas/postfix-formula/issues/50

What would you suggest?